runtime/lua/vim/lsp/health.lua

```diff
@@ -40,6 +40,7 @@ local function check_active_clients()
   local clients = vim.lsp.get_clients()
   if next(clients) then
     for _, client in pairs(clients) do
+      local server_version = vim.tbl_get(client, 'server_info', 'version') or 'n/a'
```

Comment:
similar to https://github.com/neovim/nvim-lspconfig/blob/9f2c279cf9abe584f03bfeb37c6658d68e3ff49d/lua/lspconfig/health.lua#L82 , it's good to surface a hint about what failed. that puts pressure on upstreams to fix their crap.

```suggestion
      local server_version = vim.tbl_get(client, 'server_info', 'version') or '? (no serverInfo.version response)'
```